collections/St_Johns_College/SW_V1_St_Johns_College_MS_138.xml

```diff
@@ -0,0 +1,201 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:tei="http://www.tei-c.org/ns/1.0"
+   xml:id="manuscript_12138">
+   <teiHeader xml:lang="en">
+      <fileDesc>
+         <titleStmt>
+            <title>St John's College MS. 138</title>
+            <title type="collection">St John's College MSS.</title>
+            <respStmt xml:id="RM">
+               <resp when="2002">Cataloguer</resp>
+               <persName>Ralph Hanna</persName>
+            </respStmt>
+            <respStmt xml:id="SJC">
+               <resp when="2021">Encoding</resp>
+               <persName>Sian Witherden</persName>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition>TEI P5</edition>
+            <funder>Conversion of the printed catalogue to TEI funded by <orgName>the Thompson
+                  Family Charitable Trust</orgName></funder>
+         </editionStmt>
+         <publicationStmt>
+            <publisher>Special Collections, Bodleian Libraries</publisher>
+            <address>
+               <orgName type="department">Special Collections</orgName>
+               <orgName type="unit">Bodleian Libraries</orgName>
+               <orgName type="institution">University of Oxford</orgName>
+               <street>Weston Library, Broad Street</street>
+               <settlement key="place_7011931">Oxford</settlement>
+               <postCode>OX1 3BG</postCode>
+               <country>United Kingdom</country>
+            </address>
+            <distributor>
+               <email>specialcollections.enquiries@bodleian.ox.ac.uk</email>
+            </distributor>
+            <idno type="msID">St_Johns_College_MS_138</idno>
+            <idno type="collection">St_Johns_College</idno>
+            <idno type="catalogue">Western</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:lang="en" xml:id="St_Johns_College_MS_138">
+               <msIdentifier>
+                  <country>United Kingdom</country>
+                  <region type="county">Oxfordshire</region>
+                  <settlement key="place_7011931">Oxford</settlement>
+                  <institution>St John's College</institution>
+                  <repository>St John's College</repository>
+                  <idno type="shelfmark">St John's College MS. 138</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="enm">English. Written in the language of <placeName
+                        key="place_7011980">east Cambridgeshire</placeName> (LALME LP 4265, grid
+                     568/268, forms displayed 3:30–1).</textLang>
+                  <msItem n="1" xml:id="St_Johns_College_MS_138-item1">
+                     <locus from="1r" to="126v">Fols. 1-126<hi rend="superscript">v</hi></locus>
+                     <incipit xml:lang="la">De medic<gap/> tranciat vespere decidat n<gap/> | Dauid
```

Comment:
The | symbol can be replaced by the `<lb/>` element - but it's probably only worth doing this when it falls in the middle of a word and the character would interfere with search, whereas the element will not